resources/views/layouts/beheer.blade.php

```diff
@@ -33,13 +33,15 @@
                             </a>
                         </li>
                     @else
-                        @if(\Illuminate\Support\Facades\Auth::user()->hasRole(Config::get('constants.Administrator')))
+                        @if(\Illuminate\Support\Facades\Auth::user()->hasRole(Config::get('constants.Administrator'),Config::get('constants.Certificate_administrator')))
```

Comment:
Dit zou een stuk mooier zijn met [policies](https://laravel.com/docs/5.5/authorization), maar dat is voor later.